jekyll/_cci2/release/releases-overview.adoc

```diff
@@ -0,0 +1,242 @@
+
+[#delivery]
+=== Delivery
+
+_Delivery_ is the act of packaging code changes and making them available for Deployment. Continuous delivery is the prerequisite step for continuous deployment. With some variations on the technologies being used, the delivery process creates executables from code and then makes them available from a repository to be deployed to an environment at a subsequent time.
+
+[#deployment]
+=== Deployment
+
+_Deployment_ is the act of putting a new component version into a particular environment, regardless of whether users and other services immediately interact with the new version or a previous one. Depending on the type of deployment, a release may happen as a later task (such as switching over traffic shaping rules for a blue/green deployment) or as a direct consequence of the deployment (such as a standard Kubernetes rolling update).
+
+[#release]
+=== Release
+
+A _release_ is the act of updating a component to a new version in a specific environment, causing it to become available to an audience of users and other services.
+
+A release can happen when an existing component is updated or when the first version for a new component is deployed.
+
+In the releases dashboard, releases are tagged as **Progressive** if they refer to an Argo Rollout.
+
+A release starts when the component version is updated, and ends when the new version has reached 100% availability, and all associated validations have been completed. In the case of a progressive release, this happens when the Rollout completes. In the case of a Deployment, this happens when the Deployment replicas are all available and ready.
+
+[#component]
+=== Component
+
+A _component_ in CircleCI is a collection of code and configuration that is deployed and released as a single unit. In Kubernetes terms, this would be a Deployment or Rollout object along with the related objects such as Pods, ReplicaSets, etc. that share a common `app` label.
+
+[#command]
+=== Command
+
+A command is a user-initiated action CircleCI performs on the user's behalf to manipulate a specific component. These actions are run asynchronously and CircleCI reports the results back to the CircleCI web app releases dashboard, as you see for an individual job step.
+
+Some commands which are available for all components. These are:
+
+* Restore version
+* Scale version
+* Restart version
+
+A subset of commands are available for _progressive_ releases (when using Argo Rollouts). These are:
+
+* Retry release
+* Promote release
+* Cancel release
+
+[#release-status]
+== Release status
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|Status
+|Notes
+
+|RUNNING
+| Currently in progress
+
+|FAILED
+| Pods have reached an unhealthy status. This can be recovered once the release has completed
+
+| SUCCESS
+| The Deployment or Rollout has all desired pods available
+|===
+
+[#known-limitations]
+== Known limitations
```

Comment:
@ryanbarkley I added these sections - known limitations and troubleshooting. Not sure if we have content to go here yet though

Reply:
It's a good question; I'm not sure right off but will ask the team.

Reply:
@avalcepina just started https://circleci.atlassian.net/wiki/spaces/DP/pages/6982139947/Customer+Support+Troubleshooting+Guide for internal troubleshooting docs.

Reply:
One current limitation is that if you install Argo Rollouts after installing our agent, you need to restart our agent.

Reply:
Just circling back, this is no longer a limitation:
> if you install Argo Rollouts after installing our agent, you need to restart our agent.